nautobot_chatops/dispatchers/slack.py

```diff
@@ -400,7 +409,13 @@ def multi_input_dialog(self, command, sub_command, dialog_title, dialog_list):
 
                 blocks.append(self._input_block(action_id, dialog["label"], textentry, dialog.get("optional", False)))
 
-        return self.send_blocks(blocks, callback_id=callback_id, modal=True, ephemeral=False, title=dialog_title)
+        return self.send_blocks(
+            blocks,
+            callback_id=callback_id,
+            modal=True,
+            ephemeral=settings.PLUGINS_CONFIG["nautobot_chatops"]["send_all_messages_private"],
```

Comment:
Similar concern/question here - I think a modal=True dialog is implicitly ephemeral/private in Slack, but not sure if `ephemeral=True` will actually work here or if it's an API error.

Reply:
I went ahead and moved back to False all together. You are right that it should just be part of it.